### PR TITLE
Make the port range for Stepd configurable

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**

### CODE REVIEW

File: lib/flight_scheduler/configuration.rb [hunk 1]
```diff
@@ -60,12 +62,37 @@ class Configuration
       {
         name: :node_type,
         env_var: true
+      },
+      {
+        name: :lower_ephemeral_port,
```
Comment (benarmston) [inline lib/flight_scheduler/configuration.rb]: Why is this named "ephemeral"? Wouldn't something like "stepd_port_range_start" be clearer?

**Review by benarmston — CHANGES_REQUESTED** [review]
Can we get the new configuration values added to the config file too.

File: lib/flight_scheduler/configuration.rb (hunk 1 above)
Comment (WilliamMcCumstie) [reply lib/flight_scheduler/configuration.rb]: https://github.com/openflighthpc/flight-scheduler-daemon/pull/28/commits/c8e8bf4c58af16b8eb58d6c1ac0a5d4f55d18dd4

**Comment by benarmston** [comment]
I've fixed some issues with the config loading caused by inconsistent symbol vs string usage in the various different ways that a configuration value can be loaded. See https://github.com/openflighthpc/flight-scheduler-daemon/pull/28/commits/c449c9b7bd10a296e6eefe30b1513b427ef0791f

With that done, this LGTM.

**Pull request merged**